include/sempr/entity/spatial/Collection.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef SEMPR_ENTITY_SPATIAL_COLLECTION_HPP_
+#define SEMPR_ENTITY_SPATIAL_COLLECTION_HPP_
+
+#include <sempr/entity/spatial/Geometry.hpp>
+
+namespace sempr { namespace entity {
+
+namespace geom = geos::geom;
+
+/**
+    Interface for geometry collection entities like a GeometryCollection or MultiPoint
+*/
+#pragma db object
+class Collection : public Geometry {
+    SEMPR_ENTITY
+public:
+    using Ptr = std::shared_ptr<Collection>;
+
```

Comment:
I think the "Collection"-Interface should have a getter:
```c++
const geom::GeometryCollection* geometry() const;
```
to have at least read-only access without downcasting to the more specific classes.

Reply:
The next commit is coming ... :-)

Reply:
Fixed in dcadddeb596ea4d3c0ae96020c2bbeefad3de695